### PR TITLE
Feature/calculator logic

### PR DESCRIPTION
**Pull request by unorjikingsley**
## I implemented the following in this PR: 

  > - Created a logic folder for operate and calculator.js
  > - click event handlers using the math logic from calculate.js and operate.js

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,196 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (anagudelogu) [inline README.md]: - [ ] You did a good job working on your project's documentation 💯 However, please make sure that your links are working! currently, this link to the project's issues page is broken, kindly fix it. 🤓 🔧

File: src/Components/Keypad.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  percentageLabel: PropTypes.string.isRequired,
+  divideLabel: PropTypes.string.isRequired,
+};
+
+// Keypad.defaultProps = {
+//   clearLabel: undefined,
+//   negateLabel: undefined,
+//   percentageLabel: undefined,
+//   divideLabel: undefined,
+// };
```
Comment (anagudelogu) [inline src/Components/Keypad.js]: - [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @unorjikingsley, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,70 +1,196 @@
...
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```
Comment (brainconnect93) [inline README.md:141]: - [ ] [OPTIONAL] please kindly remove all the placeholder text in all the heading of your readme file As they are there to describe what you need to do in each heading of the Readme page. Kindly remove them all.

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @unorjikingsley, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title and summary. ✔️
- Has an `operate.js` file copied from [this one](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js). ✔️
- Has a `calculate.js` file copied from [this one](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/calculate.js).✔️
- `click events` are implemented for all buttons. ✔️
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**